README.md

```diff
@@ -1,104 +1,159 @@
-# nucypher-ts
+Full Documentation can be found [here](https://docs.threshold.network/app-development/threshold-access-control-tac).
+# Get Started with TAC
 
-Communicate with NuCypher network from browser.
```

Comment:
This line is not up to date with what `nucypher-ts` is doing, nevertheless, I would prefer to have a brief one-line description of what this project does in README.